lib/helium/version.rb

```diff
@@ -1,3 +1,3 @@
 module Helium
-  VERSION = "0.12.0"
+  VERSION = "0.13.0"
```

Comment:
it doesn't matter for this pr, but we should probably bump the version in a separate commit right before release. that way if we had multiple prs coming in, they wouldn't have competing version numbers. or if we wanted to roll multiple prs into a single release we could